app/helpers/pipeline_outputs_helper.rb

```diff
@@ -140,6 +140,18 @@ def taxon_name(taxid, tax_level)
     taxon["#{tax_level}_name"] if taxon
   end
 
+  def get_taxid_fasta_from_pipeline_run_combined_nt_nr(pipeline_run, taxid, tax_level)
+    nt_array = get_taxid_fasta_from_pipeline_run(pipeline_run, taxid, tax_level, 'NT').split(">")
+    nr_array = get_taxid_fasta_from_pipeline_run(pipeline_run, taxid, tax_level, 'NR').split(">")
+    combined_array = ((nt_array | nr_array) - [''])
```

Comment:
so are users expecting that we combine nt and nr fastas? do we document this anywhere? it seems like a detail I would want to know as a user

Reply:
Made a note to add in the docs. Thanks! In the future, we may add an option to let them pick nt or nr.